demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -1625,14 +1594,15 @@ static void prvDisconnectFromMQTTBroker( void )
     /* End TLS session, then close TCP connection. */
     ( void ) SecureSocketsTransport_Disconnect( &networkContextMqtt );
 }
+/*-----------------------------------------------------------*/
 
-static int32_t connectToS3Server( NetworkContext_t * pNetworkContext,
-                                  const char * pUrl )
+static int32_t connectToS3Server( NetworkContext_t * pxNetworkContext,
+                                  const char * pcUrl ) /*todo fixfunction */
 {
-    int32_t returnStatus = EXIT_SUCCESS;
-    HTTPStatus_t httpStatus = HTTPSuccess;
+    BaseType_t returnStatus = pdPASS;
+    HTTPStatus_t xHTTPStatus = HTTPSuccess;
```

Comment:
Maybe change this to "xHttpStatus" to be consistent with the xMqttStatus variables that you have.

Reply:
Updated in https://github.com/aws/amazon-freertos/pull/3148/commits/6bbe8a3c217e31fff82104a685ad811120104724